modules/stac/stac-collection/providers.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    http = {
+      source = "hashicorp/http"
```

Comment:
I don't think we need the http provider anymore since we're running this from a Kubernetes Job. Maybe change this whole file to

```
terraform {
  required_providers {
    kubernetes = {
      source = "hashicorp/kubernetes"
    }
  }
}
```